extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -334,14 +334,14 @@ export class SchemaCompareResult {
 		}).component();
 
 		this.generateScriptButton.onDidClick(async (click) => {
-			// get file path
+			// generate default filename
 			let now = new Date();
 			let datetime = now.getFullYear() + '-' + (now.getMonth() + 1) + '-' + now.getDate() + '-' + now.getHours() + '-' + now.getMinutes() + '-' + now.getSeconds();
-			let rootPath = vscode.workspace.rootPath ? vscode.workspace.rootPath : os.homedir();
-			let defaultFilePath = path.join(rootPath, this.targetName + '_Update_' + datetime + '.sql');
+			let defaultFileName = this.targetName + '_Update_' + datetime + '.sql';
```

Comment:
nit: But this can be more cleanly written using string interpolation

const defaultFileName = `${this.targetName}_Update_${datetime}.sql`